workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -18,10 +18,6 @@ public class Bootstrap : MonoBehaviour
 
         public void Awake()
         {
-            // Taken from DefaultWorldInitalization.cs
-            SetupInjectionHooks(); // Register hybrid injection hooks
-            PlayerLoopManager.RegisterDomainUnload(DomainUnloadShutdown, 10000); // Clean up worlds and player loop
```

Comment:
Is this no longer required?

Reply:
How do we have a world per worker without the injection hooks?

Reply:
these are just the hooks that were set up when registering the default world. As we disabled the default world, the hooks were never initialized and we had to do it on our own. Now that the default world gets created, we don't need to set them up anymore. the `DefaultWorldInitialization` takes care of that